### DISCUSSION TITLE
请问 deepin 的开发代号是什么？

### DISCUSSION THREAD

**Discussion by ykla** [linuxdeepin/developer-center#5109] [Development Questions | 开发答疑] [answered]
比如 deepin 23 是 beige

请问其他版本的代号分别是多少，能否列个表格出来？

**Comment by BLumia** [comment, marked as answer]
15：eagle
20：apricot
23：beige

注：均为社区版 codename。之前 deepin 的商业版本和社区版本都还叫 deepin 并且大版本号一样，但 codename 是不同的，在此不再列出。
